src/runtime_src/core/include/xrt_mem.h

```diff
@@ -118,6 +118,7 @@ struct xcl_bo_flags
  */
 #define XRT_BO_USE_NORMAL 0
 #define XRT_BO_USE_DEBUG  1
+#define XRT_BO_USE_KMD    2
```

Comment:
Please add brief comment as to the purpose of this new use flag.  I am asking because I already forgot why we need the flag?  I think it is something like this:  For the table BO we allocate a normal host only BO but we set the USE flag to this new `XRT_BO_USE_KMD` so that shim can take proper action when creating the allocation for the BO?

So amend the comment to something like:
```
The flag `XRT_BO_USE_KMD`  indicates that the buffer content can be shared with the kernel mode driver.  
This flag controls how the shim level constructs the allocation for the buffer.
```